crawler/crawler.py

```diff
@@ -219,14 +229,8 @@ def main():
     else:
         raise NotImplementedError('Invalid crawlmode')
 
-    emitters = EmittersManager(urls=args.url,
-                               format=args.format,
-                               compress=args.compress,
-                               extra_metadata=args.extraMetadata)
-    worker = Worker(emitters, args.frequency, crawler)
-
     try:
-        worker.run()
+        crawler.run()
```

Comment:
worker also meant crawl + possibly filter + emit. With 'crawler' instead, that separation seems to go away.